website/cue/reference/remap/functions/parse_influxdb.cue

```diff
@@ -0,0 +1,98 @@
+package metadata
+
+remap: functions: parse_influxdb: {
+	category: "Parse"
+	description: """
+		Parses the `value` as an [InfluxDB line protocol](https://docs.influxdata.com/influxdb/v1/write_protocols/line_protocol_reference/)
+		string, producing a list of Vector-compatible metrics".
+		"""
+	notices: [
+		"""
+			The only metric type that is produced is a `gauge`. Each metric name is prefixed with the `measurement` field, followed
+			by an underscore (`_`), and then the `field key` field.
+			""",
+	]
+	arguments: [
+		{
+			name:        "value"
+			description: "The string representation of the InfluxDB line protocol to parse."
+			required:    true
+			type:        ["string"]
+		},
+	]
+	internal_failure_reasons: [
+		"`value` is not a valid InfluxDB line protocol string.",
+	]
+	return: types: ["object"]
+
+	examples: [
+		{
+			title: "Parse InfluxDB line protocol"
+			source: #"""
```

Comment:
This example is the same from the vrl repo PR. 

Is is better to have a single example depicting all possible field value types (like this one), or should I split it in 3 or so,
so this example is not "bloated"?

Reply:
I think this example is ok and shows all of the possible field values well.